irohad/torii/processor/query_processor_impl.hpp

```diff
@@ -52,6 +52,8 @@ namespace iroha {
       std::shared_ptr<iroha::PendingTransactionStorage> pending_transactions_;
       std::shared_ptr<shared_model::interface::QueryResponseFactory>
           response_factory_;
+      std::shared_ptr<shared_model::interface::PermissionToString>
```

Comment:
A strange one, where is it initialized and used?

Reply:
https://github.com/hyperledger/iroha/blob/a0da9d9f5ba1952ee37a1ebcdb19fe41a3317d7d/irohad/ametsuchi/impl/storage_impl.cpp#L58
https://github.com/hyperledger/iroha/blob/a0da9d9f5ba1952ee37a1ebcdb19fe41a3317d7d/irohad/ametsuchi/impl/storage_impl.cpp#L74-L76
https://github.com/hyperledger/iroha/blob/a0da9d9f5ba1952ee37a1ebcdb19fe41a3317d7d/irohad/ametsuchi/impl/storage_impl.cpp#L102
https://github.com/hyperledger/iroha/blob/a0da9d9f5ba1952ee37a1ebcdb19fe41a3317d7d/irohad/ametsuchi/impl/storage_impl.cpp#L156